subjects/ascii-art/output/audit.md

```diff
@@ -2,17 +2,21 @@
 
 ###### Has the requirement for the allowed packages been respected? (Reminder for this project: only [standard packages](https://golang.org/pkg/))
 
+<<<<<<< Updated upstream
```

Comment:
an auto generated line from git to be removed

Reply:
fixed